src/Evolution/Systems/GeneralizedHarmonic/GaugeSourceFunctions/DampedHarmonic.hpp

```diff
@@ -115,7 +123,11 @@ void damped_harmonic_h(
     double sigma_t_L1, double t_start_L2, double sigma_t_L2, double t_start_S,
     double sigma_t_S,
     // weight function
-    double sigma_r) noexcept;
+    double sigma_r,
+    // toggle the switch below to change
+    // H = (1 - R0) H0 + mu1 T1 + mu2 T2 + muS TS, to
+    // H = (1 - R0) H0 + R0 (mu1 T1 + mu2 T2 + muS TS)
```

Comment:
move this comment to the doxygen comment.

Reply:
was already there, highlighted it now